### DISCUSSION TITLE
How to add more variants other than dark?

### DISCUSSION THREAD

**Discussion by T04435** [tailwindlabs/tailwindcss#13967] [Help] [answered]
Hi need some help.

In my app I have multiple themes: dark, light, foo, bar

I want to apply styles like

```
// does not work 
<div className="rounded-md foo:rounded-lg" />
// works
<div className="rounded-md dark:rounded-lg" />

```

is this possible?

**Comment by wongjn** [comment, marked as answer]
Yes! Consider [registering some custom variants in a Tailwind plugin](https://tailwindcss.com/docs/plugins#static-variants).
